pandas/tests/test_panel.py

```diff
@@ -1528,6 +1528,12 @@ def test_fillna(self):
         p.iloc[0:2, 0:2, 0:2] = np.nan
         self.assertRaises(NotImplementedError, lambda: p.fillna(999, limit=1))
 
+    def test_fillna_inplace(self):
+        panel = self.panel.copy()
```

Comment:
you are not actually testing whether this worked or not because there aren't any NA's in the panel!.